src/internal/observable/range.ts

```diff
@@ -25,7 +25,7 @@ import { Observable } from '../Observable';
  *
  * @param {number} [start=0] The value of the first integer in the sequence.
  * @param {number} [count=0] The number of sequential integers to generate.
- * @param {Scheduler} [scheduler] A `Scheduler` to use for scheduling
+ * @param {Scheduler} [scheduler] A {@link SchedulerLike} to use for scheduling
```

Comment:
The `{Scheduler}` here should be `{SchedulerLike}` . (same elsewhere in this commit)

Reply:
The types of the params should not be necessary. I believe that the dgeni processing will extract this info from the function declaration below. You should try it and see.

Reply:
It looks like when listing overloads it reads function declarations and prints proper `"Type: xyz"` but "Parameters" section at the top of the page usually doesn't mention any type. I'm not sure why is that.